### ISSUE TITLE
Yahoo Price Data - Python Scraper

### ISSUE THREAD

**Issue by jleacox** [TheWallStreetAnalytics/wallstreetanalytics#23]
Found this code a while back, can't recall where - but it works!
[fetch.zip](https://github.com/TheWallStreetAnalytics/wallstreetanalytics/files/6008156/fetch.zip)

**Comment by Objective-Resolve** [comment]
There is something similar for R that can be found [here](https://www.r-bloggers.com/2017/03/an-introduction-to-stock-market-data-analysis-with-r-part-1/). It's a multipart series. I can go through it and assemble it into one file if that would be helpful. I am more familiar with R then Python, but I could also try doing it with Python (the person who wrote the R script does it with Python too). Do we want to use Yahoo or Google or both for financial data?

**Comment by jleacox** [comment]
> There is something similar for R that can be found [here](https://www.r-bloggers.com/2017/03/an-introduction-to-stock-market-data-analysis-with-r-part-1/). It's a multipart series. I can go through it and assemble it into one file if that would be helpful. I am more familiar with R then Python, but I could also try doing it with Python (the person who wrote the R script does it with Python too). Do we want to use Yahoo or Google or both for financial data?

Yeah after I posted this I saw there's also already a Yahoo scraper elsewhere on this repo, so just more fuel for the fire ;)
